src/nextjs/nextjs-wizard.ts

```diff
@@ -189,6 +191,84 @@ export async function runNextjsWizardWithTelemetry(
     }
   });
 
+  await traceStep('create-global-error-page', async () => {
+    const maybeAppDirPath = path.join(process.cwd(), 'pages');
+    const maybeSrcAppDirPath = path.join(process.cwd(), 'src', 'pages');
+
+    const appDirLocation =
+      fs.existsSync(maybeAppDirPath) &&
+      fs.lstatSync(maybeAppDirPath).isDirectory()
+        ? ['app']
+        : fs.existsSync(maybeSrcAppDirPath) &&
+          fs.lstatSync(maybeSrcAppDirPath).isDirectory()
+        ? ['src', 'app']
+        : undefined;
+
+    if (!appDirLocation) {
+      return;
+    }
+
+    const globalErrorPageFile = fs.existsSync(
+      path.join(process.cwd(), ...appDirLocation, 'global-error.tsx'),
+    )
+      ? 'global-error.tsx'
+      : fs.existsSync(
+          path.join(process.cwd(), ...appDirLocation, 'global-error.ts'),
+        )
+      ? 'global-error.ts'
+      : fs.existsSync(
+          path.join(process.cwd(), ...appDirLocation, 'global-error.jsx'),
+        )
+      ? 'global-error.jsx'
+      : fs.existsSync(
+          path.join(process.cwd(), ...appDirLocation, 'global-error.js'),
+        )
+      ? 'global-error.js'
+      : undefined;
+
+    if (!globalErrorPageFile) {
+      await fs.promises.writeFile(
+        path.join(process.cwd(), ...appDirLocation, 'global-error.jsx'),
+        getSentryDefaultGlobalErrorPage(),
+        { encoding: 'utf8', flag: 'w' },
+      );
+
+      clack.log.success(
+        `Created ${chalk.bold(
```

Comment:
l: should we use `chalk.cyan` here instead? I think we mostly use it when mentioning file names. But not sure how consistent we are with it in Nextjs, so feel free to disregard

Reply:
oh you're right. Will update.